f2/src/summary/LocationInfoSummary.js

```diff
@@ -1,36 +1,58 @@
 /** @jsx jsx */
+import React from 'react'
 import { jsx } from '@emotion/core'
 import { Trans } from '@lingui/macro'
-import { Stack, Flex, Code, Alert, AlertIcon } from '@chakra-ui/core'
+import { Stack, Flex } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
 import { testdata, EditButton } from '../ConfirmationSummary'
 import { H2 } from '../components/header'
+import { DescriptionListItem } from '../components/DescriptionListItem'
+import { Text } from '../components/text'
 
-export const LocationInfoSummary = props => {
+export const LocationInfoSummary = ({ onSubmit }) => {
   const [data] = useStateValue()
-
-  const locationInfo = {
-    ...data.formData.locationInfo,
-    ...testdata.formData.locationInfo, //Remove after done testing
+  const location = {
+    ...testdata.formData.location, //Remove after done testing
+    ...data.formData.location,
   }
 
+  const hasInfoToDisplay =
+    location.postalCode.length > 0 || location.cityTown.length > 0
+
   return (
-    <Stack spacing={4} borderBottom="2px" borderColor="gray.300" pb={4}>
-      <Flex align="baseline">
-        <H2>
-          <Trans id="confirmationPage.LocationInfoTitle" />
-        </H2>
-        <EditButton path="/location" label="Edit affected business assets" />
```

Comment:
Needs a label in relation to the path. 

Also needs to be translated, but this is another battle